tailwind.config.ts

```diff
@@ -13,6 +13,9 @@ const config: Config = {
         "gradient-conic":
           "conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))",
       },
+      colors: {
+        'campaign-background': '#c2c2c2',
```

Comment:
create scss variable file seperately and define color code there so we can use it at multiple places